### ISSUE TITLE
QN feature request: more human-readable representation for JOY amounts

### ISSUE THREAD

**Issue by traumschule** [Joystream/joystream#4515]
Add field `amountHuman` (or similar) with ',' as more human-readable representation for all JOY amounts.

**Comment by mochet** [comment]
Would this mean the value is presented as human readable and ignore decimals while also ensuring users who want decimals are accommodated for?

**Comment by traumschule** [comment]
Could be as simple as adding the dot:
```
query {
    rewardPaymentEvents {
    inBlock
    createdAt
    paidBalance
    councilMember { member {handle}}
  } }

        "inBlock": 374401,
        "createdAt": "2023-01-04T21:43:00.007Z",
        "paidBalance": "55555555550400",
        "paidBalanceHuman": "5555.5555550400",
        "councilMember": {
          "member": {
            "handle": "abramaria_"
          }
```
If my math is correct that's about $333 per day (at original 6ct/JOY):
> 55555555550400 / 10000000000 * 0.06 $/JOY = $333.33333330240004

**Comment by mochet** [comment]
My question is whether we want to change the QN to suite queries by general users in the absence of there being no great general tools available?
I am not suggesting the issue is incorrect but in the grand scheme of things It seems more prudent to provide better general tools to people vs expecting them to actually utilize the QN directly.

TLDR: If better tools were available in the first place, I presume there would be very few users putting manual queries into QNs

**Comment by mochet** [comment]
<img width="350" alt="image" src="https://user-images.githubusercontent.com/50830161/211705089-ae307b41-6503-419e-a5ec-be1deedf5fc1.png">
<img width="398" alt="image" src="https://user-images.githubusercontent.com/50830161/211705172-bcbfcce1-267f-4f23-a0b9-d848a994a172.png">


It appears that some of the returned values may not be consistent? But I'm really not sure.

Query used was:
`query MyQuery {
  workersConnection {
    edges {
      node {
        id
        rewardPerBlock
      }
    }
  }
}
`

**Comment by traumschule** [comment]
From https://github.com/Joystream/joystream/issues/4503#issuecomment-1379996587
>  would either require a lot of mapping updates or some clever Hydra changes

Wonder if this could even be solved with a UI tweak when hovering amounts.
